File: framework/languages/java/samples/java/TicTacToe.SessionGateway/src/main/java/systems/zlink/samples/tictactoe/sessiongateway/shared/configuration/SampleNames.java
package systems.zlink.samples.tictactoe.sessiongateway.shared.configuration;

public final class SampleNames {
    public static final String ApiChannel = "tictactoe-session-api";
    public static final String PlayChannel = "tictactoe-session-play";
    public static final String SpotMesh = "tictactoe";
    public static final String PlayRouteChannel = "play-route";
    public static final String PlayNode = "play";
    public static final String SessionRelayNode = "session-relay";
    public static final String GatewayStream = "gateway";
    public static final String PlayerActorType = "player";
    public static final String EntrySpotRoutingId = "3301";
    public static final String TurnChangedPacket = "TurnChangedNotify";
    public static final String OpponentJoinedPacket = "OpponentJoinedNotify";
    public static final String GameEndedPacket = "GameEndedNotify";

    private SampleNames() {
    }
}

File: framework/languages/java/samples/kotlin/TicTacToe.SessionGateway/src/main/kotlin/systems/zlink/samples/kotlin/tictactoe/sessiongateway/shared/configuration/SampleNames.kt
package systems.zlink.samples.kotlin.tictactoe.sessiongateway.shared.configuration

object SampleNames {
    const val ApiChannel: String = "tictactoe-session-api"
    const val PlayChannel: String = "tictactoe-session-play"
    const val SpotMesh: String = "tictactoe"
    const val PlayRouteChannel: String = "play-route"
    const val PlayNode: String = "play"
    const val SessionRelayNode: String = "session-relay"
    const val GatewayStream: String = "gateway"
    const val PlayerActorType: String = "player"
    const val EntrySpotRoutingId: String = "3301"
    const val TurnChangedPacket: String = "TurnChangedNotify"
    const val OpponentJoinedPacket: String = "OpponentJoinedNotify"
    const val GameEndedPacket: String = "GameEndedNotify"
}
